cypress-tests/cypress/support/commands.js

```diff
@@ -2033,6 +2056,7 @@ Cypress.Commands.add("createJWTToken", (req_data, res_data, globalState) => {
       expect(response.body).to.have.property("token");
       //set jwt_token
       globalState.set("jwtToken", response.body.token);
+      globalState.set("merchantId", response.body.merchant_id);
```

Comment:
Are these needed? These are already setup, right?

Reply:
@pixincreate This is related to routing. I need to get the publishable key for the merchant which we created manually in the dashboard. This is the way i get get the merchant id and i can retrieve the merchant for that.

Reply:
Cool. Got it. Can you please add a comment in that code at least. So that, we'll know in future.  
Asking so because, we already set that up above. Since we have a reason to set it here, I feel there is a need to have clarification as to why we're doing it.